tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot

```diff
@@ -33,3 +33,94 @@ Connect to non-existent network
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect cross1-network2 ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+
+#    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} create --name cross1-container2 busybox ping -c2 cross1-container
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect cross1-network ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect cross1-network2 ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} logs --follow cross1-container2
+#    Should Be Equal As Integers  ${rc}  0
+#    Should Contain  ${output}  2 packets transmitted, 2 packets received
+
+#Connect containers to multiple networks non-overlapping
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network create cross2-network
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network create cross2-network2
+#    Should Be Equal As Integers  ${rc}  0
+
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} create --name cross2-container busybox /bin/top
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect cross2-network ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+
+#    ${rc}  ${containerID}=  Run And Return Rc And Output  docker ${params} create --name cross2-container2 busybox ping -c2 cross2-container
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} network connect cross2-network2 ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${containerID}
+#    Should Be Equal As Integers  ${rc}  0
+#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} logs --follow cross2-container2
+#    Should Be Equal As Integers  ${rc}  0
+#    Should Not Contain  ${output}  2 packets transmitted, 2 packets received
+
+#Connect containers to multiple networks non-overlapping with a bridge container
```

Comment:
This test as it is laid out should not work. The "bridge" container should be able to ping the other two containers. The other two containers should not be able to reach each other since they are on separate networks.